src/theory/quantifiers/sygus/term_database_sygus.cpp

```diff
@@ -515,6 +560,33 @@ void TermDbSygus::registerEnumerator(Node e,
     }
   }
   Trace("sygus-db") << "  ...finished" << std::endl;
+
+  d_enum_var_agnostic[e] = isVarAgnostic;
+  if (isVarAgnostic)
+  {
+    // if not done so already, compute type class identifiers for each variable
```

Comment:
where else could that have been computed?

Reply:
If this is the second enumerator of this type, we will have already computed this information.